src/main/java/org/gridsuite/securityanalysis/server/dto/SecurityAnalysisAdditionalParametersInfos.java

```diff
@@ -0,0 +1,29 @@
+/**
+ Copyright (c) 2023, RTE (http://www.rte-france.com)
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.securityanalysis.server.dto;
+
+import com.powsybl.loadflow.LoadFlowParameters;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili@rte-france.com>
+ */
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class SecurityAnalysisAdditionalParametersInfos {
```

Comment:
We should keep the same name as in StudyServer I think 
https://github.com/gridsuite/study-server/blob/f7dcc3478622291e2fe5d27249d092fc10fc96b3/src/main/java/org/gridsuite/study/server/dto/LoadFlowParametersInfos.java#L21